source/common/config/grpc_mux_impl.cc

```diff
@@ -17,7 +17,11 @@ GrpcMuxImpl::GrpcMuxImpl(const LocalInfo::LocalInfo& local_info, Grpc::AsyncClie
       service_method_(service_method), random_(random), time_source_(dispatcher.timeSystem()),
       control_plane_stats_(generateControlPlaneStats(scope)) {
   Config::Utility::checkLocalInfo("ads", local_info);
+
+  // Bucket contains 100 tokens maximum and refills at 10 tokens/sec.
+  limit_request_ = std::make_unique<TokenBucketImpl>(100, time_source_, 10);
```

Comment:
@htuch Since this rate limit is applied for all api types, I have increased it to 10 per sec. Please review.

Reply:
I think we should make this configurable at some point. Maybe for now make these named constants, and leave a TODO to plumb this into `ApiConfigSource` unless you want to do this now.

Reply:
Agree. Was thinking the same. will leave a TODO for now